test/functional/testExternalScripts.md

```diff
@@ -0,0 +1,6 @@
+The external script [MathJax 2.75](https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML) should be included, and the following MathJax content should render:

```

Comment:
These files shouldn't be outside? They are supposed to be inside `test_site`.

EDIT: `test_site` already has these files, so delete this one.

Reply:
My bad I think something went wrong during the rebase process